package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "todoliststructure",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack --config webpack.config.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "css-loader": "^6.7.1",
+    "style-loader": "^3.3.1",
+    "webpack": "^5.70.0",
+    "webpack-cli": "^4.9.2"
+  },
+  "dependencies": {
+    "html-webpack-plugin": "^5.5.0"
+  }
```

Comment:
- [ ] according to the requirements  Webpack dev server should be set up and running with no errors.

Reply:
Any ScreenShot of Errors that you are talking about?

Reply:
Hi Rao,
I just put the entire requirement in this comment, what I'm aiming at is the fact that you don't have "webpack-dev-server" setup at all

Reply:
Ohhh. OK. That was the issue, I was unable to understand. Thanks for the suggestion.